### DISCUSSION TITLE
Presión de los neumáticos

### DISCUSSION THREAD

**Discussion by egpall** [CarlosCuezva/dashboards-Grafana-Teslamate#13] [Q&A]
Hola Carlos, acabo de descubrir esto de tus Dashboards y es una pasada, ya tenía funcionando Teslamate y Grafana.
El Dashboard de la presión de los neumáticos no funciona, se podría arreglar? 
Gracias por todo tu trabajo

**Comment by CarlosCuezva** [comment]
Hola!!

Me alegro que te gusten. ¿Me podrías dar más pistas con el dashboard de las presiones? ¿Qué falla, qué error te da,...?

Muchas gracias

**Reply by egpall** [reply]
![F1D8A6CE-D8B3-491F-BBD2-18D5C78FAE64](https://user-images.githubusercontent.com/61494891/224648972-043343b9-ecb6-4f87-a25b-ca25bfda3ea3.png)

Gracias a ti!!!

**Comment by CarlosCuezva** [comment]
¿Qué versión de Teslamate tienes instalada? 

He estado mirando y esa funcionalidad la añadieron en la versión 1.27.0 y el coche tiene que disponer la versión en el coche 2022.4 o superior

**Reply by egpall** [reply]
1.26.1 en Docker en un Qnap, si te digo la verdad no sé cómo actualizarlo, luego me pondré y lo intentaré

**Comment by CarlosCuezva** [comment]
Tienes que hacerlo desde el terminal SSH, normalmente el docker-compose.yaml se guarda en 

/share/CACHEDEV1_DATA/Container/container-station-data/application/teslamate

y allí sigue las instrucciones de Teslamate

https://docs.teslamate.org/docs/upgrading

**Reply by egpall** [reply]
Es que no lo tengo en compose, sino individual en Container Station.

**Comment by CarlosCuezva** [comment]
Ummm sino has usado la opción "Create application" entonces ya si que ni idea de cómo actualizarlo

**Reply by egpall** [reply]
Sí, lo hice con Create Application

**Comment by CarlosCuezva** [comment]
Entonces si que te genera el solo un docker-compose y te sería fácil actualizarlo

**Reply by egpall** [reply]
Conseguido!!!! He seguido un vídeo de Juan Vidal Haces de Baleares. Todos los Docker se actualizan así? Estoy muy verde. Gracias por toda la ayuda.

**Comment by CarlosCuezva** [comment]
Oleeeeeee felicidades!!!

Todos los que estén mediante docker-compose básicamente si. Conozco a Juan jejej
